tests/unit/filter-step-form.spec.ts

```diff
@@ -20,12 +20,12 @@ describe('Filter Step Form', () => {
       data: {
         editedStep: {
           name: 'filter',
-          condition: { column: 'toto', operator: 'eq', value: '' },
+          condition: { column: 'toto', operator: 'eq', value: {} },
```

Comment:
OK! IMO, we could have a more relevant test on this validation.
With the widgets available in the forms, I don't see any case where users can send an object as value (`value: {}`)
If you want to keep a validation test, I suggest testing the column name instead (it shouldn't be empty) https://github.com/ToucanToco/weaverbird/pull/569/files#diff-41a18ffe8fcf0208fad725aaee286fd9R91-R93

Reply:
Perfect, let's go for it!